vhdbuilder/packer/components.json

```diff
@@ -224,11 +218,9 @@
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/tigera/operator:*",
-      "amd64OnlyVersions": [
-        "v1.20.0"
-      ],
+      "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.22.3",
```

Comment:
the 1.22.3 is not in use?

Reply:
1.22.3 is the first multi-arch I plan to use, but we backfilled the 1.20.0 currently in use to 'multi-arch'.